recipes/eagle2/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.4.1" %}
+
+package:
+  name: eagle2
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('eagle2', max_pin='x.x.x') }}
+
+source:
+  url: https://github.com/poruloh/Eagle/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 85804bfe972186ccb66e602856d6e04044302e5c0dbf9309849b0cde981e3432
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - htslib >=1.9
+    - bzip2
+    - zlib
+    - boost-cpp >=1.58
+    - openblas >=0.2.19
+  run:
+    - {{ pin_compatible('htslib') }}
+    - {{ pin_compatible('bzip2') }}
+    - {{ pin_compatible('boost-cpp') }}
+    - {{ pin_compatible('zlib') }}
+    - {{ pin_compatible('openblas') }}
```

Comment:
```suggestion
```

This is usually not needed, because those packages define run_exports on their own.

Reply:
Okay I will delete them

Reply:
If I remove them the boost library is not found.
I've looked in the bioconda repository and people seems to run the last four of them (they are not part of bioconda).